### DISCUSSION TITLE
First time chatter

### DISCUSSION THREAD

**Discussion by Vikkio92** [TwitchIO/TwitchIO#313] [Feature requests]
Maybe I just haven't been able to find it, but I think it would be useful to be able to tell when a viewer is a first time chatter.

I assume it is triggered when they chat for the first time so it is a property of a message rather than a user, but I am not a programmer so I have no idea really.

I just thought it might have not been included in the library since it's a relatively new addition to the platform.

**Comment by chillymosh** [comment]
This will be addressed in our next release, but it will be Message.first

**Comment by IAmTomahawkx** [comment]
This has been released in version 2.5
